src/chrome/content/code/HTTPSRules.js

```diff
@@ -472,8 +519,9 @@ const HTTPSRules = {
     }
   },
 
-
-  rewrittenURI: function(alist, input_uri) {
+  // return true iff callback has been called already, false if callback will be
+  // called asynchronously
```

Comment:
There is a comment a few lines below that documents a different, now-obsolete return value.